chia-wallet/src/proof.rs

```diff
@@ -29,13 +30,13 @@ impl<'a> Arbitrary<'a> for Proof {
         let is_eve = u.ratio(3, 10)?;
         if is_eve {
             Ok(Self::Eve(EveProof {
-                parent_coin_info: u.arbitrary()?,
+                parent_coin_info: (u.arbitrary::<[u8; 32]>()?).into(),
                 amount: u.arbitrary()?,
             }))
         } else {
             Ok(Self::Lineage(LineageProof {
-                parent_coin_info: u.arbitrary()?,
-                inner_puzzle_hash: u.arbitrary()?,
+                parent_coin_info: (u.arbitrary::<[u8; 32]>()?).into(),
+                inner_puzzle_hash: (u.arbitrary::<[u8; 32]>()?).into(),
```

Comment:
does `Bytes32` not implement `Arbitrary`? (I think it should)

Reply:
`Bytes32` *does* implement `Arbitrary`, but only when the `fuzzing` feature is enabled.
I imagine you really only need the `Arbitrary` implementation when fuzzing as well